RetailCoder.VBE/App.cs

```diff
@@ -39,10 +40,10 @@ public class App : IDisposable
         private readonly IConnectionPoint _projectsEventsConnectionPoint;
         private readonly int _projectsEventsCookie;
 
-        private readonly IDictionary<VBProjectsEventsSink, Tuple<IConnectionPoint, int>>  _componentsEventsConnectionPoints = 
-            new Dictionary<VBProjectsEventsSink, Tuple<IConnectionPoint, int>>();
-        private readonly IDictionary<VBProjectsEventsSink, Tuple<IConnectionPoint, int>> _referencesEventsConnectionPoints =
-            new Dictionary<VBProjectsEventsSink, Tuple<IConnectionPoint, int>>();
```

Comment:
This was outright wrong; connectionpoints need to be per-project, yet the key `VBProjectsEventsSink` has a `VBProjects` connection point, which lives at application-level - in other words this was never going to be working for more than 1 project.